corehq/apps/locations/models.py

```diff
@@ -107,13 +111,20 @@ def _populate_stock_levels(self):
         self.overstock_threshold = config.overstock_threshold
 
     def save(self, *args, **kwargs):
+        from .tasks import sync_administrative_status
         if not self.code:
             from corehq.apps.commtrack.util import unicode_slug
             self.code = unicode_slug(self.name)
         if not self.commtrack_enabled:
             self.administrative = True
         self._populate_stock_levels()
-        return super(LocationType, self).save(*args, **kwargs)
+        saved = super(LocationType, self).save(*args, **kwargs)
+
+        if self._administrative_old != self.administrative:
```

Comment:
`self._administrative_old` isn't set until after the `super` call to `__init__`, so it will already be set to either True or False:
```python
In [1]: lt = LocationType(domain='ethan-commtrack', name='foo')  

In [2]: lt.administrative  
Out[2]: False
```
That means it won't register a difference unless `administrative` is changed outside of the constructor (which could happen).  Actually, what we could do is add `is_first_save = self.pk is None` right before the actual save call, then skip the task if that's `True`.